kedro-airflow/tests/conftest.py

```diff
@@ -99,14 +98,6 @@ def register_pipelines():
 @fixture(scope="session")
 def metadata(kedro_project):
     # cwd() depends on ^ the isolated filesystem, created by CliRunner()
-    project_path = kedro_project
-    return ProjectMetadata(
-        source_dir=project_path / "src",
-        config_file=project_path / "pyproject.toml",
-        package_name="hello_world",
-        project_name="Hello world !!!",
-        kedro_init_version=kedro_version,
-        project_path=project_path,
-        tools=["None"],
-        example_pipeline="No",
-    )
+    project_path = kedro_project.resolve()
+    metadata = bootstrap_project(project_path)
+    return metadata
```

Comment:
Is this now necessary for the test setup because we don't call `bootstrap_project` in the actual flow anymore?

Reply:
Yeah, since we don't do it in the plugin code, it's necessary for the unit tests to run.